src/views/AdminViews/Signup/SignupView.vue

```diff
@@ -0,0 +1,403 @@
+<template>
+  <BaseLayout id="base-layout">
+    <CoverComponent id="cover"></CoverComponent>
+    <div class="signup-container">
+      <div class="content">
+        <!-- Form -->
+        <form @submit.prevent="app.submitForm">
+          <div class="content-title">
+            <h1 class="title">Create your account</h1>
+            <span>Already have an account? <router-link to="" class="link">Sign in here</router-link></span>
+          </div>
+
+          <div class="signup-google">
+            <button type="button" class="g-btn google-btn" @click.prevent="">
+              <span class="item">
+                <img class="avatar" src="@\assets\img\svg\google.svg" alt="Google Icon" />
+                Sign up with Google
+              </span>
+            </button>
+          </div>
+
+          <div class="content-or">
+            <span class="divider text-muted">OR</span>
+          </div>
+
+          <!-- Form Group -->
+          <div class="form-group">
+            <label class="input-label" for="organizationName">Organization name</label>
+
+            <input
+              type="text"
+              :class="['input-form', { 'is-invalid': false, 'is-valid': false }]"
+              name="name"
+              id="organizationName"
+              placeholder="Organization name"
+              v-model="app.organizationName.value"
+              required
+            />
+
+            <div class="invalid-feedback" v-if="false">Please enter your Organization name.</div>
+          </div>
+          <!-- End Form Group -->
+
+          <!-- Form Group -->
+          <div class="form-group">
+            <label class="input-label" for="organizationEmail">Organization email</label>
+
+            <input
+              type="email"
+              :class="['input-form', { 'is-invalid': false, 'is-valid': false }]"
```

Comment:
{ 'is-invalid': false, 'is-valid': false } fix cứng bằng false thì cho vào làm gì thế a